lib/Test/MockFile.pm

```diff
@@ -457,7 +457,8 @@ sub dir {
     _validate_path($dir_name);
 
     # Cleanup trailing forward slashes
-    $dir_name =~ s{[/\\]$}{}xmsg;
+    $dir_name ne '/'
```

Comment:
New question. Is it even sane to allow mocking of the / dir?

Reply:
I'll find the test that did this and show you. We can throw a fit/exception over it, too.